Cargo.toml

```diff
@@ -14,7 +14,7 @@ exclude = ["/.github", "/.dependabot", ".gitignore"]
 [dependencies]
 tokio = { version = "0.2.22", features = ["full"]}
 kube = "0.43.0"
-k8s-openapi = { version = "0.9.0", features = ["v1_15"], default-features = false }
+k8s-openapi = { version = "0.9.0", features = ["v1_16"], default-features = false }
```

Comment:
does upgrade to v1_16 required ?

Reply:
It is required for sepc.overhead to be defined.